insights_messaging/consumers/kafka.py

```diff
@@ -22,6 +23,8 @@ def __init__(
         config = kwargs.copy()
         config["group.id"] = group_id
         config["bootstrap.servers"] = ",".join(bootstrap_servers)
+        if kwargs.get("group.instance.id"):
+            config["group.instance.id"] = os.environ.get("HOSTNAME")
```

Comment:
*if* it's ok to set this to _something_ by default I'd go with:
```
config["group.instance.id"] = kwargs.get("group.instance.id", os.environ.get("HOSTNAME"))
```

Reply:
that makes sense to me. I'll give that go